libs/wingsdk/src/target-sim/service.ts

```diff
@@ -46,11 +63,59 @@ export class Service extends cloud.Service implements ISimulatorResource {
   }
 
   public onLift(host: IInflightHost, ops: string[]): void {
-    bindSimulatorResource(__filename, this, host);
+    bindSimulatorResource(__filename, this, host, ops);
     super.onLift(host, ops);
   }
 
   public _toInflight(): string {
     return makeSimulatorJsClient(__filename, this);
   }
 }
+
+/** @internal */
+export const SERVICE_HELPER_FQN = fqnForType("sim.ServiceHelper");
+
+/** @internal */
+export interface ServiceHelperProps {
+  readonly service: Service;
+  readonly autoStart: boolean;
+}
+
+/**
```

Comment:
This doesn't feel right and makes me wonder if modeling policy as a resource is the right choice. Why not just add a `policy` attribute in the simulator schema to any inflight host (compute) resource (service, function, on-deploy) directly?

Not sure I fully understand the value of modeling policies as resources. What's the motivation?

Reply:
https://github.com/winglang/wing/pull/6079#issuecomment-2023734113